forge-game/src/main/java/forge/game/trigger/TriggerRevealed.java

```diff
@@ -7,6 +7,7 @@
 import forge.game.spellability.SpellAbility;
 import forge.util.Localizer;
 
+@Deprecated
```

Comment:
Why is the Trigger Deprecated when its still in use of TriggerType?

Reply:
There‘s no card that makes use of it

Reply:
yeah its for old Miracle stuff, i think the Trigger can be removed